CMakeLists.txt

```diff
@@ -353,91 +356,112 @@ if (OPENAL)
   endif()
 endif()
 
+set(BASE_DATA_DIR "./" CACHE INTERNAL "Base data dir")
 
-if (NOT APPLE)
-#install(TARGETS barony
-#  RUNTIME DESTINATION bin
-#  COMPONENT Runtime
-#)
+if (NOT APPLE AND UNIX)
+include(GNUInstallDirs)
+if (DATA_DIR)
+	set(BASE_DATA_DIR "${CMAKE_INSTALL_FULL_DATAROOTDIR}/barony")
+endif()
+message(STATUS "Base data directory ${BASE_DATA_DIR}")
+
+install(TARGETS barony
+	RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+  COMPONENT Runtime
+)
 else()
 install(TARGETS barony
   RUNTIME DESTINATION bin COMPONENT Runtime BUNDLE DESTINATION Resources
 )
 endif()
 
+add_definitions(-DBASE_DATA_DIR=\"${BASE_DATA_DIR}\")
+
 if (APPLE)
   #add_executable(editor OSX/SDLmain.m ${EDITOR_SOURCES})
   #set(MACOSX_BUNDLE_ICON_FILE "${PROJECT_SOURCE_DIR}/editor.icns")
   #add_executable(editor MACOSX_BUNDLE OSX/SDLmain.m ${EDITOR_SOURCES} ${COPY_FRAMEWORKS} libfmodex.dylib  /opt/local/lib/libpng16.16.dylib)
   #add_executable(editor MACOSX_BUNDLE OSX/SDLmain.m ${EDITOR_SOURCES} libfmodex.dylib  /opt/local/lib/libpng16.16.dylib)
   if (FMOD_ENABLED)
-    add_executable(editor MACOSX_BUNDLE ${EDITOR_SOURCES} libfmodex.dylib  /opt/local/lib/libpng16.16.dylib)
+	  add_executable(${EDITOR_EXE_NAME} MACOSX_BUNDLE ${EDITOR_SOURCES} libfmodex.dylib  /opt/local/lib/libpng16.16.dylib)
```

Comment:
The indentation should be all spaces or all tabs, let's not mix them. My preference would be 2 spaces as this seems to be the most common standard for cmake, but I don't care that much as long as it's consistent. @addictgamer ?